tests/unit/data/base/test_image.py

```diff
@@ -0,0 +1,34 @@
+"""Unit Tests - Base Image Datamodules."""
+
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from anomalib.data import AnomalibDataModule
+import random
+from .test_datamodule import _TestAnomalibDataModule
+
+
+class _TestAnomalibImageDatamodule(_TestAnomalibDataModule):
+    def test_get_item_returns_correct_shapes(self, datamodule: AnomalibDataModule) -> None:
+        """Test that the datamodule __getitem__ returns image, mask, label and boxes."""
+
+        # Randomly select a subset of the dataset.
+        subsets = ["train", "val", "test"]
```

Comment:
Why is this random? We won't be able to check if bounding boxes and masks are returned when it is the train set